docs/olmv1_mvp.md

```diff
@@ -0,0 +1,136 @@
+# OLM v1 MVP
```

Comment:
The functional/behavioral requirements included in this doc are actually beyond the scope of the MVP.  What do you all think if we title this as "OLM v1 Project Requirement Document (PRD)"?





Tony Wu
  [5 minutes ago](https://kubernetes.slack.com/archives/C0181L6JYQ2/p1681442993466969?thread_ts=1681333000.404779&cid=C0181L6JYQ2)
if we want to swap out Product Requirement Document (PRD) to something less business focused, should we call it something like “project requirement document”? (edited) 





Tony Wu
  [4 minutes ago](https://kubernetes.slack.com/archives/C0181L6JYQ2/p1681443027065769?thread_ts=1681333000.404779&cid=C0181L6JYQ2)
this means its coverage is beyond the MVP scope (edited) 





Tony Wu
  [4 minutes ago](https://kubernetes.slack.com/archives/C0181L6JYQ2/p1681443044380709?thread_ts=1681333000.404779&cid=C0181L6JYQ2)
and the bonus point is the shorthand is still “PRD” (edited) 





Tony Wu
  [4 minutes ago](https://kubernetes.slack.com/archives/C0181L6JYQ2/p1681443051958609?thread_ts=1681333000.404779&cid=C0181L6JYQ2)
what do you all think?


Tony Wu
  [1 minute ago](https://kubernetes.slack.com/archives/C0181L6JYQ2/p1681443223709359?thread_ts=1681333000.404779&cid=C0181L6JYQ2)
let me add that to the PR too

Reply:
Maybe "OLMv1 Roadmap"? IMO it would be nice to break this down into different sections that include what is needed for the MVP and then what is planned for the future. Maybe a structure like:

```md
# OLM v1 Roadmap

## MVP

### Functional Requirements
### Behavioral Requirements

## Future Work

### Functional Requirements
### Behavioral Requirements
```

That way we can have a clear line between what is necessary for the MVP and what is not.

Reply:
I've changed the title and the filename to "roadmap" in this PR, and should we defer defining the MVP and Future Work scope in the future PR?